src/index.scss

```diff
@@ -1,24 +1,31 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900;1000&display=swap');
 
-$blue: #465491;
+$dark-blue: #1B315E;
+$blue: #2A3E7F;
+$light-blue: #455296;
+$red: #F1646B;
+$orange: #F9A36A;
+$pink: #F9C3BC;
+$beige: #F4ECE0;
 
-/*
-tablet & mobile have the column layout
-tablet has fixed widths and font sizes
-mobile has scaling widths and font sizes
-*/
+$assets: './bh7-assets/';
 
-$desktop-width: 1210px;
+$desktop-width: 1200px;
 $mobile-width: 750px;
+$mobile-ratio: 75vh; // width must be >= 1.33 * height (surface duo and vertical ipad are exactly 1.33x)  
 
 @mixin desktop {
 	@media(min-width: #{$desktop-width}) { @content; }
 }
-/*tablet mixin also applies to mobile*/
-@mixin tablet {
-	@media(max-width: #{$desktop-width}) { @content; }
-}
 @mixin mobile {
+	@media(max-width: #{$desktop-width}) { 
+		@media(max-width: #{$mobile-ratio}) {
+			@content;
+		}
+	}
+}
+// unused, tentative
+@mixin small-mobile {
```

Comment:
Is this more for width-restricted or height-restricted mobile?

Reply:
Currently, it's not used at all. This is the old mobile media query.